content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -111,13 +113,8 @@ with the ability to create suitably secure and isolated Pods, you should enforce
 You can use [Pod Security admission](/docs/concepts/security/pod-security-admission/)
 or other (third party) mechanisms to implement that enforcement.
 
-You can also use the deprecated [PodSecurityPolicy](/docs/concepts/security/pod-security-policy/) mechanism
-to restrict users' abilities to create privileged Pods (N.B. PodSecurityPolicy is scheduled for removal
-in version 1.25).
-
-Creating a workload in a namespace also grants indirect access to Secrets in that namespace. 
-Creating a pod in kube-system or a similarly privileged namespace can grant a user access to 
-Secrets they would not have through RBAC directly.
+For these reasons, it is considered best practice to generally grant permissions at the namespace
+level, and use caution when subdividing access within a namespace.
```

Comment:
```suggestion
For these reasons, as a best practice, grant permissions at the namespace
level and use caution when subdividing access within a namespace.
```
It's not clear what "subdividing access" means. What specifically should they be cautious about? What configuration could lead to issues?

Reply:
How's this?

> For these reasons, it is considered best practice to generally grant permissions at the namespace
level, and use caution when attempting to limit access to a subset of namespace resources.